code/modules/mob/living/carbon/carbon.dm

```diff
@@ -208,7 +208,11 @@
 	loc.handle_fall(src)//it's loc so it doesn't call the mob's handle_fall which does nothing
 
 /mob/living/carbon/is_muzzled()
-	return(istype(src.wear_mask, /obj/item/clothing/mask/muzzle))
+	/// if this is bad runtime, then I'm sorry. I don't know how to make it both universal and efficient.
```

Comment:
My logic, here, is that you could theoretically imagine an item on most clothing categories (except for, like, glasses and boots) which blocks speech. I want to be able to account for those eventualities, in case someone else wants to create an item which blocks speech.

Reply:
It's fine, but removing `is_muzzled` and creating an element for this is much better. I won't require it now, but it's something we should do eventually. Remove the comment.